curriculum/challenges/english/blocks/workshop-bookstore-page/69134d14150d51f8066a3246.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 69134d14150d51f8066a3246
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+Now add the `<body>` tag. This is where all of your visible page content will go.
+
+# --hints--
+
+Your code should include an `<body>` tag.
+
+```js
+assert.match(code, /<body>/i);
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>XYZ Bookstore Page</title>
```

Comment:
`lang` attribute and `meta` elements are not asked for in step 1, they can't appear like this, you need to ask and test for them